### DISCUSSION TITLE
How does “kmer” this parameter influence complete chloroplast and how can I find the most suitable parameter according to log when I failed to get complete sequence of chloroplast ?

### DISCUSSION THREAD

**Discussion by curtainDian** [Kinggerm/GetOrganelle#177] [Q&A]
Hi ！ Mr. Kinggerm . I want to ask a question about the influence of kmer to the result of distangling fastg?
Nearly all of my complete results are based on the kmer which I setted the largest such as 127 or 115. However，less than 7% of my complete results are based on the middle one. （I frequently set -k 21，45，65，85，105，127）
So how does "kmer" influence the result that whether chloroplast can be circularized ? I worked it for my tutor , and I can not explain it very clearly to him.
And when I failed to get complete result , the sentences appear in my log file are frequently either "Incomplete/Complicated graph: please check around EDGE" or "Multiple isolated embplant_pt components detected! Broken or contamination?". I read the FAQ about the solution, but I want to know whether I can judge which parameter is the most crucial according to the log file? 
Thanks a lot , Mr. Kinggerm
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/8990076/get_org.log.txt)
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/8990078/get_org.log.txt)

**Comment by JianjunJin** [comment]
Please check our [GetOrganelle paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02154-5) for the kmer question. Specifically, see the `k-mer in GetOrganelle` part.

See the following answer for the second question.
https://github.com/Kinggerm/GetOrganelle/wiki/FAQ#what-should-i-do-with-incomplete-resultbroken-assembly-graph
The solution comes case by case. The assembly graph is also important to check.

**Reply by curtainDian** [reply]
Thanks a lot ! I will try it.
